### ISSUE TITLE
Found the error go.opentelemetry.io/otel/sdk/export for opentelemetry with jaeger exporter

### ISSUE THREAD

**Issue by iamgoangle** [open-telemetry/opentelemetry-go#470]
I am trying to run the example https://github.com/open-telemetry/opentelemetry-go/blob/master/example/jaeger/main.go

trying to get all dependencies
![image](https://user-images.githubusercontent.com/6595022/74107462-eced9600-4ba2-11ea-8c81-ea7f52a36f61.png)

I got an error like this
![image](https://user-images.githubusercontent.com/6595022/74105859-34b8f100-4b94-11ea-9e45-36620535707c.png)

Would you please help? This is my repositoty https://github.com/iamgoangle/opentelemetry-jaeger-exporter

**Comment by wingyplus** [comment]
Seems like it fetch jaeger exporter as a v1.0.0.

```
$ go get -v go.opentelemetry.io/otel/exporter/trace/jaeger
go: finding versions for go.opentelemetry.io/otel/exporter/trace
go: finding versions for go.opentelemetry.io/otel/exporter
go: finding versions for go.opentelemetry.io
go: go.opentelemetry.io/otel/exporter/trace/jaeger upgrade => v1.0.0
go: finding versions for go.opentelemetry.io/otel/sdk/export
go: finding versions for go.opentelemetry.io/otel/sdk
../../../../pkg/mod/go.opentelemetry.io/otel/exporter/trace/jaeger@v1.0.0/jaeger.go:27:2: module go.opentelemetry.io/otel@latest found (v0.2.1), but does not contain package go.opentelemetry.io/otel/sdk/export
```

Not sure why it's upgrade to v1.0.0. So I just try the workaround by mark a version to v0.2.1 during go get jaeger exporter

```go
$ go get -v go.opentelemetry.io/otel/exporter/trace/jaeger@v0.2.1
```

**Comment by wingyplus** [comment]
It's seems to be a bad release on golang module proxy. Try it with `GOPROXY=direct` is work fine.

```
$ env GOPROXY=direct sh try.sh
go: creating new go.mod: module github.com/wingyplus/repro
get "go.opentelemetry.io/otel": found meta tag get.metaImport{Prefix:"go.opentelemetry.io/otel", VCS:"git", RepoRoot:"https://github.com/open-telemetry/opentelemetry-go"} at //go.opentelemetry.io/otel?go-get=1
go: finding versions for go.opentelemetry.io/otel
go: go.opentelemetry.io/otel upgrade => v0.2.1
can't load package: package go.opentelemetry.io/otel: build constraints exclude all Go files in /Users/wingyplus/pkg/mod/go.opentelemetry.io/otel@v0.2.1
get "go.opentelemetry.io/otel": found meta tag get.metaImport{Prefix:"go.opentelemetry.io/otel", VCS:"git", RepoRoot:"https://github.com/open-telemetry/opentelemetry-go"} at //go.opentelemetry.io/otel?go-get=1
get "go.opentelemetry.io/otel/exporter/trace": found meta tag get.metaImport{Prefix:"go.opentelemetry.io/otel", VCS:"git", RepoRoot:"https://github.com/open-telemetry/opentelemetry-go"} at //go.opentelemetry.io/otel/exporter/trace?go-get=1
get "go.opentelemetry.io/otel/exporter/trace": verifying non-authoritative meta tag
get "go.opentelemetry.io/otel/exporter/trace/jaeger": found meta tag get.metaImport{Prefix:"go.opentelemetry.io/otel", VCS:"git", RepoRoot:"https://github.com/open-telemetry/opentelemetry-go"} at //go.opentelemetry.io/otel/exporter/trace/jaeger?go-get=1
get "go.opentelemetry.io/otel/exporter/trace/jaeger": verifying non-authoritative meta tag
get "go.opentelemetry.io/otel/exporter": found meta tag get.metaImport{Prefix:"go.opentelemetry.io/otel", VCS:"git", RepoRoot:"https://github.com/open-telemetry/opentelemetry-go"} at //go.opentelemetry.io/otel/exporter?go-get=1
get "go.opentelemetry.io/otel/exporter": verifying non-authoritative meta tag
go: finding versions for go.opentelemetry.io/otel
go: finding versions for go.opentelemetry.io/otel/exporter/trace/jaeger
go: finding versions for go.opentelemetry.io/otel/exporter
go: finding versions for go.opentelemetry.io/otel/exporter/trace
go: finding go.opentelemetry.io/otel/exporter latest
go: finding go.opentelemetry.io/otel/exporter/trace latest
go: go.opentelemetry.io/otel/exporter/trace/jaeger upgrade => v0.2.1
$ cat try.sh
go mod init
go get -v go.opentelemetry.io/otel
go get -v go.opentelemetry.io/otel/exporter/trace/jaeger
```

**Comment by rghetia** [comment]
@krnowak suggested to rename exporter to exporters. This will affect all exporters but it is okay since it is not GA.

**Comment by rghetia** [comment]
will be addressed by #331
